spec/lib/recog/nizer_spec.rb

```diff
@@ -165,4 +289,23 @@
 
   end
 
+
+  describe '.load_db' do
+    file_path = File.expand_path(File.join('spec', 'data', 'test_fingerprints.xml'))
+    context "with #{file_path}" do
+      let(:fp_db) { subject.load_db(file_path) }
+      it "loads without error" do
+        expect(fp_db).to  be true
+        subject.unload_db()
+      end
+    end
+
+    context "with invalid file path" do
+      it "raises an error" do
+        expect { subject.load_db('no_such_987_file_path') }.to raise_error(Errno::ENOENT)
+        subject.unload_db()
+      end
+    end
```

Comment:
Went ahead an added an explicit test for calling w/o no parameters because.. stuff happens..